nixos/modules/services/web-servers/garage.nix

```diff
@@ -51,11 +51,11 @@ in
             default = "none";
             type = types.enum ([ "none" "1" "2" "3" 1 2 3 ]);
             apply = v: toString v;
-            description = lib.mdDoc "Garage replication mode, defaults to none, see: <https://garagehq.deuxfleurs.fr/reference_manual/configuration.html#replication_mode> for reference.";
+            description = lib.mdDoc "Garage replication mode, defaults to none, see: <https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/#replication-mode> for reference.";
           };
         };
       };
-      description = lib.mdDoc "Garage configuration, see <https://garagehq.deuxfleurs.fr/reference_manual/configuration.html> for reference.";
+      description = lib.mdDoc "Garage configuration, see <https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/> for reference.";
```

Comment:
An extraneous white space here.

Reply:
where?

Reply:
Ah, blame my browser's poor font rendering.
![screenshot-2023-04-06-18-46-23](https://user-images.githubusercontent.com/15247171/230355177-de4828dc-d87a-49dc-84a0-fa8395daf070.png)

Reply:
that's a scary thing, ooh!
how did you end up with font rendering that broken?

Reply:
Strange ligatures of jetbrains mono and whatever frontend thing that goes beyond my understanding.